.github/actions/build_and_test/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Build and Test
+
+runs:
+  using: composite
+  steps:
+    - name: Execute Tests
+      shell: bash
+      run: |
+        bazelisk test //... 
+
+    - name: Build Anything Not Tested
```

Comment:
doesn't this happen in the `test` command since we're not passing `--build_tests_only`?